index.html

```diff
@@ -510,6 +519,55 @@ <h3>Security and Privacy Considerations</h3>
+
+				<p>For an audiobook, the User Agent should provide a <a href="https://www.w3.org/TR/pwp-ucr/#player">player interface</a> that will allow the user to navigate, play, or pause the audiobook. This interface can be represented to the user in any way (i.e. physical buttons, visual interface, keyboard input, or voice commands), but should be accessible at any point in the listening experience.</p>
+
+			</section>
+
+			<section id="audio-ua-packaging">
+				<h3>Audiobook Packaging and Offlining</h3>
+
+				<p>The <a href="https://www.w3.org/TR/pwp-ucr">Use Cases and Requirements</a> note recommends that content be available offline and that any packaged formats should not affect the iterations of the publications. This means that even if the content is copied many times to many users via multiple User Agents, the core manifest and it's identifier are never changed.</p>
+
+				<p>This specification recommends the <a href="https://www.w3.org/TR/lpf">Lightweight Packaging Format</a> for packaging audiobook content, but this is a requirement. Audiobook User Agents should be able to ingest LPF files for play, and should display content according to the requirements and recommendations in this document.</p>
+
+				<p>If a User Agent is serving the content directly from their service (i.e. as a retailer or repository of content), it is recommended that they provide a method for offlining or downloading the content to the user. This can be in any format they choose, but the audiobook should be complete and valid and the contents listed in the manifest should be served in their entirety. Even if a User Agent does not support the display of a certain resource (i.e. an image file or data table), it should still be available to the user for download.</p>
+
+				<p>This specification does not provide a method for content creators to protect or watermark their content, as there are existing methods available in the market today. User Agents who work with content creators that wish to protect or limit the distribution of their content can choose a method that works best for their requirements.</p>
+
+			</section>
+
+			<section id="audio-ua-accessibility">
+				<h3>Audiobooks Accessibility</h3>
+
+				<p>This specification recommends and provides a method for content creators to create fully accessible audiobooks. User Agents should use this information, in the section on <a href="#audio-accessiblity">Accessibility</a>, to implement accessible audiobook interfaces. Depending on the limitations of the user agent, it is recommended that User Agents provide accessible player interfaces, as well as a method for content creators who have provided <code>alternate</code> content to have that content displayed.</p>
```

Comment:
The link to the accessibility section should probably be in the first sentence so the recommended method is clear.

I'd avoid potentially contradictory statements like this:

> Depending on the limitations of the user agent, it is recommended...

Just recommend the practice. A failure to follow the recommendation is baked into it not being a "must" and/or this being an informative section.